### PR TITLE
Feature/comments

### PR DESCRIPTION
**Pull request by marcelblanarik**

### CODE REVIEW

**Comment by TomTasche** [comment]
looks good in a first test on mobile now, but desktop seems weird now (text doesn't span the whole page)

**Comment by marcelblanarik** [comment]
i made a little change in css, please check now

File: index.html [hunk 1]
```diff
@@ -102,7 +102,7 @@
 
 .pagedjs_page {
 		display: block;
-    width: calc(100 / 23 * 21vw)!important;
+    width: calc(100 / 23 * 21)!important;
```
Comment (TomTasche) [inline index.html:101]: what's the difference?

File: index.html [hunk 2]
```diff
@@ -119,19 +119,18 @@
...
-    width: 100%;
+    width: 100%!important;
     height: calc(100 / 23 * 29.7)!important;
     margin: calc(100 / 23 * 1vw) auto!important;
   }
 
   .pagedjs_page_content, .pagedjs_area, .pagedjs_pagebox, .pagedjs_sheet, .content, .pagedjs_margin .pagedjs_margin-bottom-center .hasContent, .pagedjs_margin-content, span, .Normal {
-    width: 100vw!important;
-    text-align: justify!important;
+    width: 90vw!important;
```
Comment (TomTasche) [inline index.html:124]: should we change it back to 100 now? does the old code come from an official sample? if so, would be great to keep our code as similar as possible to make it easier to maintain in the future

**Review by TomTasche — CHANGES_REQUESTED** [review]
works as expected now! a few more comments to make sure we stay as close as possible to the official sample code.

moreover, I've decided that it would be better to remove the page count because it's misleading (page numbers would not represent the real page numbers as seen in LibreOffice). please make that change too

PS: please link to the sample code you used to create this

File: index.html (hunk 2 above)
Comment (marcelblanarik) [reply index.html:124]: when I change it back to 100, it covers 100% of the width, so there is no free space from left and right. 
also, I'm not sure what you mean by official sample, did you find more than this? https://www.pagedjs.org/documentation/ unfortunately they don't mention responsivity at all (at least I didn't find it)

File: index.html (hunk 1 above)
Comment (marcelblanarik) [reply index.html:101]: it was my mistake, vw is relative to 1% of the width of the viewport, that's why it was so wide before, now it should represent A4 page size more or less

File: index.html (hunk 2 above)
Comment (TomTasche) [reply index.html:124]: oh I thought you copied some of the CSS from somewhere? if not that's fine of course

**Comment by TomTasche** [comment]
one last thing I noticed: there seems to be too much margin on the right side? the margin on both sides should be equal in my opinion

**Comment by marcelblanarik** [comment]
> one last thing I noticed: there seems to be too much margin on the right side? the margin on both sides should be equal in my opinion

changed it, please check

**Pull request merged**